seml/__main__.py

```diff
@@ -0,0 +1,678 @@
+                 "Disable this if the check fails unexpectedly when using "
+                 "advanced Sacred features or to accelerate adding.",
+        ),
+    ] = True,
+    code_checkpoint: Annotated[
+        bool,
+        typer.Option(
+            help="Save a checkpoint of the code in the database. "
+                    "Disable this if you want your experiments to use the current code"
+                    "instead of the code at the time of adding.",
+        ),
+    ] = True,
+    force: Annotated[
+        bool,
+        typer.Option(
+            help="Force adding the experiment even if it already exists in the database.",
+        ),
+    ] = False,
+    overwrite_params: Annotated[
+        dict,
+        typer.Option(
+            help="Dictionary (passed as a string, e.g. '{\"epochs\": 100}') to overwrite parameters in the config.",
+            metavar='JSON',
+            parser=json.loads
+        ),
+    ] = None,
+):
+    """
+    Add experiments to the database as defined in the configuration.
+    """
+    assert ctx.obj['collection'], "Collection name must be passed to this command."
+    add_config_files(
+        ctx.obj['collection'],
+        config_files,
+        force_duplicates=force,
+        no_hash=not hash,
+        no_sanity_check=not sanity_check,
+        no_code_checkpoint=not code_checkpoint,
+        overwrite_params=overwrite_params
+    )
+
+@app.command("start")
+def start_command(
+    ctx: typer.Context,
+    sacred_id: SacredIdAnnotation = None,
+    filter_dict: FilterDictAnnotation = None,
+    batch_id: BatchIdAnnotation = None,
+    debug: Annotated[
+        bool,
+        typer.Option(
+            '-d',
+            '--debug',
+            help="Run a single interactive experiment without Sacred observers and with post-mortem debugging. "
+                 "Implies `--verbose --num-exps 1 --post-mortem --output-to-console`.",
+        ),
+    ] = False,
+    debug_server: Annotated[
+        bool,
+        typer.Option(
+            '-ds',
+            '--debug-server',
+            help="Run the experiment with a debug server, to which you can remotely connect with e.g. VS Code. "
+                 "Implies `--debug`.",
+        ),
+    ] = False,
+    local: Annotated[
+        bool,
+        typer.Option(
+            '-l',
+            '--local',
+            help="Run the experiment locally instead of on a Slurm cluster.",
+        ),
+    ] = False,
+    no_worker: Annotated[
+        bool,
+        typer.Option(
+            '-nw',
+            '--no-worker',
+            help="Do not launch a local worker after setting experiments' state to PENDING.",
+        ),
+    ] = False,
+    num_exps: NumExperimentsAnnotation = 0,
+    file_output: FileOutputAnnotation = True,
+    steal_slurm: StealSlurmAnnotation = False,
+    post_mortem: PostMortemAnnotation = False,
+    output_to_console: OutputToConsoleAnnotation = False,
```

Comment:
We lose `-o` as a shorthand. Maybe add that to the `OutputToConsoleAnnotation` annotation?